### DISCUSSION TITLE
How to manage allure tests?

### DISCUSSION THREAD

**Discussion by SkYMaaN** [allure-framework/allure2#2589] [Questions & Support] [answered]
I ran the same testsfew times.  The results were generated in the 'allure-results' directory. But when I use the 'allure serve' command, I can only see the report for the most recently run tests ( 6 / 12 ). How can I put the results of all the tests (previously run) in one report?
![image](https://github.com/allure-framework/allure2/assets/71893955/cccdf5e9-86df-4977-9149-094a2090a234)
![image](https://github.com/allure-framework/allure2/assets/71893955/a2b1f131-6b29-473d-b99a-e513283e80de)

**Comment by baev** [comment, marked as answer]
Allure only shows the latest result per test. For the results that have had retries (previous results), you can see the retry mark. To review the previous test result, navigate to the test result page and select the "Retries" tab. 

For more details, please see https://allurereport.org/docs/history-and-retries/
